features/AuthController.feature

```diff
@@ -19,6 +19,32 @@ Feature: Auth Controller
       | valid | false      |
       | state | "_STRING_" |
 
+  @security @http
+  Scenario: Check if a token in the cookie is valid
+    Given I send a HTTP "POST" request with:
+      | controller | "auth"                                               |
+      | action     | "login"                                              |
+      | strategy   | "local"                                              |
+      | body       | { "username": "test-admin", "password": "password" } |
```

Comment:
Nitpicking: you can construct object "on the fly"
```suggestion
      | body.username       | "test-admin"  |
      | body.password        | "password"  |
```

Reply:
Same as above, It seems that I cannot construct "object on the fly"